assets/js/main.js

```diff
@@ -156,13 +156,35 @@ $(document).ready((function ($) {
         return false;
     });
 
+    $('.appointment-button').click(function () {
+        var $form = $('.appointment-form'),
+            data = getFormData($form),
+            host = 'http://127.0.0.1:8000/',
+            path = 'api/appointment/',
+            url = host + path;
+        $.ajax({
+            type: 'POST',
+            url: url,
+            data: data,
+            success: function () {
```

Comment:
You're missing here `beforeSend: removeFieldErrors,` - see how you did before.

Because of that, errors from the appointment HTML form are not cleared if a valid field value is submitted.

![MROOM 2022-01-12 23-10-55](https://user-images.githubusercontent.com/12160621/149222156-fdbaa2ae-a7cd-4ce2-8f44-45767afdad12.png)

-------

Also modify `removeFieldErrors` function:
![mroomapp – main js 2022-01-12 23-13-48](https://user-images.githubusercontent.com/12160621/149222347-919125dc-6988-4724-b670-0e60251439e6.png)

Reply:
Done

Reply:
Resolved.